src/fights.ts

```diff
@@ -348,10 +347,11 @@ export function dailyFights(): void {
       if (have($familiar`Pocket Professor`) && !get<boolean>("_garbo_meatChain", false)) {
         const startLectures = get("_pocketProfessorLectures");
         useFamiliar($familiar`Pocket Professor`);
-        meatOutfit(true, [
-          ...fightSource.requirements,
-          new Requirement([], { forceEquip: $items`Pocket Professor memory chip` }),
-        ]);
+        const requirement = Requirement.merge(fightSource.requirements);
+        const forceEquip = requirement.maximizeOptions.forceEquip ?? [];
+        forceEquip.push($item`Pocket Professor memory chip`);
+        requirement.maximizeOptions.forceEquip = forceEquip;
```

Comment:
Does this work? `maximizeOptions` is a getter function, not the property itself.

Reply:
yeah